readthedocs/projects/constants.py

```diff
@@ -383,43 +383,40 @@
     'https://bitbucket.org/{user}/{repo}/'
     'src/{version}{docroot}{path}{source_suffix}'
 )
-BITBUCKET_COMMIT_URL = (
-    'https://bitbucket.org/{user}/{repo}/'
-    'commits/{commit}'
-)
+BITBUCKET_COMMIT_URL = "https://bitbucket.org/{user}/{repo}/commits/{commit}"
 GITLAB_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    '{action}/{version}{docroot}{path}{source_suffix}'
-)
-GITLAB_COMMIT_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'commit/{commit}'
+    "https://gitlab.com/{user}/{repo}/"
+    "{action}/{version}{docroot}{path}{source_suffix}"
 )
+GITLAB_COMMIT_URL = "https://gitlab.com/{user}/{repo}/commit/{commit}"
 GITLAB_MERGE_REQUEST_COMMIT_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'commit/{commit}?merge_request_iid={number}'
-)
-GITLAB_MERGE_REQUEST_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'merge_requests/{number}'
+    "https://gitlab.com/{user}/{repo}/commit/{commit}?merge_request_iid={number}"
 )
+GITLAB_MERGE_REQUEST_URL = "https://gitlab.com/{user}/{repo}/merge_requests/{number}"
 
 # Patterns to pull merge/pull request from providers
-GITHUB_PR_PULL_PATTERN = 'pull/{id}/head:external-{id}'
-GITLAB_MR_PULL_PATTERN = 'merge-requests/{id}/head:external-{id}'
+GITHUB_PR_PULL_PATTERN = "pull/{id}/head:external-{id}"
+GITLAB_MR_PULL_PATTERN = "merge-requests/{id}/head:external-{id}"
 
 # Git provider names
-GITHUB_BRAND = 'GitHub'
-GITLAB_BRAND = 'GitLab'
+GITHUB_BRAND = "GitHub"
+GITLAB_BRAND = "GitLab"
 
 # SSL statuses
-SSL_STATUS_VALID = 'valid'
-SSL_STATUS_INVALID = 'invalid'
-SSL_STATUS_PENDING = 'pending'
-SSL_STATUS_UNKNOWN = 'unknown'
+SSL_STATUS_VALID = "valid"
+SSL_STATUS_INVALID = "invalid"
+SSL_STATUS_PENDING = "pending"
+SSL_STATUS_UNKNOWN = "unknown"
 SSL_STATUS_CHOICES = (
-    (SSL_STATUS_VALID, _('Valid and active')),
-    (SSL_STATUS_INVALID, _('Invalid')),
-    (SSL_STATUS_PENDING, _('Pending')),
-    (SSL_STATUS_UNKNOWN, _('Unknown')),
+    (SSL_STATUS_VALID, _("Valid and active")),
+    (SSL_STATUS_INVALID, _("Invalid")),
+    (SSL_STATUS_PENDING, _("Pending")),
+    (SSL_STATUS_UNKNOWN, _("Unknown")),
+)
+
+SINGLE_VERSION_WITHOUT_TRANSLATIONS = "single_version_without_translations"
+MULTIPLE_VERSIONS_WITH_TRANSLATIONS = "multiple_versions_with_translations"
+VERSIONING_SCHEME_CHOICES = (
```

Comment:
This PR looks pretty much done to me, but we still don't have "Single version with translations" or similar in this list. I thought that was part of this PR?

Reply:
That's going to be added in https://github.com/readthedocs/readthedocs.org/pull/10766